### ISSUE TITLE
Errors with latest FDPS version

### ISSUE THREAD

**Issue by gassmoeller** [NatsukiHosono/FDPS_SPH#12]
Just for reference, I see the following errors when compiling with the latest (of today) FDPS version. If this will be fixed in a future version anyway then this issue can be closed.
```
In file included from /home/rene/software/FDPS_SPH/FDPS/src/particle_simulator.hpp:6:0,
                 from /home/rene/software/FDPS_SPH/src/main.cpp:1:
/home/rene/software/FDPS_SPH/FDPS/src/particle_system.hpp: In instantiation of ‘void ParticleSimulator::ParticleSystem<Tptcl>::readParticleImpl(const char*, const char*, Theader*, void (Tptcl::*)(FILE*), ParticleSimulator::S32 (Theader::*)(FILE*), const char*) [with Theader = FileHeader; Tptcl = STD::RealPtcl; FILE = _IO_FILE; ParticleSimulator::S32 = int]’:
/home/rene/software/FDPS_SPH/FDPS/src/particle_system.hpp:281:29:   required from ‘void ParticleSimulator::ParticleSystem<Tptcl>::readParticleAscii(const char*, const char*, Theader&) [with Theader = FileHeader; Tptcl = STD::RealPtcl]’
/home/rene/software/FDPS_SPH/src/io.h:43:2:   required from ‘void InputFileWithTimeInterval(ParticleSimulator::ParticleSystem<Ptcl>&, system_t&) [with ThisPtcl = STD::RealPtcl]’
/home/rene/software/FDPS_SPH/src/main.cpp:42:64:   required from here
/home/rene/software/FDPS_SPH/FDPS/src/particle_system.hpp:167:38: error: ‘class STD::RealPtcl’ has no member named ‘readBinary’
                             ptcl_tmp.readBinary(fp);
                             ~~~~~~~~~^~~~~~~~~~
CMakeFiles/sph.out.dir/build.make:62: recipe for target 'CMakeFiles/sph.out.dir/src/main.cpp.o' failed
make[2]: *** [CMakeFiles/sph.out.dir/src/main.cpp.o] Error 1
CMakeFiles/Makefile2:67: recipe for target 'CMakeFiles/sph.out.dir/all' failed
make[1]: *** [CMakeFiles/sph.out.dir/all] Error 2
Makefile:83: recipe for target 'all' failed
make: *** [all] Error 2
```

**Comment by mikinakajima** [comment]
Hi - I just tried the FDPS_SPH-master version and it did work on my mac. But I had to change a few things:
This is for my Mac:
In Makefile

CC = g++-mp-8

I also moved init/GI.h to GI.h, and change the main.cpp accordingly. Also in GI.h I changed <parse.h> to "parse.h"

I know these are minor changes but it can be a problem for a beginner like me :D 
I am going to try the same thing with Rene's EOS version. Also, I am trying to compile on Linux on my cluster... I will keep you posted.
